contrib/completion/bash/docker

```diff
@@ -1142,6 +1142,29 @@ __docker_complete_user_group() {
 	fi
 }
 
+PLUGINS_PATH=$(docker info --format '{{range .ClientInfo.Plugins}}{{.Path}},{{end}}')
```

Comment:
Two things here:

- this defines a global variable (accessable in the current shell). Maybe its name should be more specific.
- `,` as a path list delimiter looks a bit unexpected to me, would you mind using a `:` for that purpose?
```suggestion
DOCKER_PLUGINS_PATH=$(docker info --format '{{range .ClientInfo.Plugins}}{{.Path}}:{{end}}')
```

(Corresponding change in https://github.com/docker/cli/blob/5c3a594bcbc414c86e47d8fc32d2729d6ee564d9/contrib/completion/bash/docker#L5492 required)